Client/src/components/cities.js

```diff
@@ -0,0 +1,83 @@
+import React, {Component} from 'react';
```

Comment:
I think the comments are important for your own notes but maybe it makes sense to separate them out of the file.  There's a lot going on and I think the formatting in general needs to be adjusted a bit.